sn_interface/src/messaging/auth_kind.rs

```diff
@@ -35,17 +35,17 @@ pub enum AuthKind {
     ///
     /// Section share authority is needed for messages related to section administration, such as
     /// DKG and relocation.
-    NodeBlsShare(BlsShareAuth),
+    SectionPart(SectionAuthPart),
```

Comment:
I'd keep the `Share` name here too.
```suggestion
    SectionShare(SectionAuthPart),
```